framework/web/Response.php

```diff
@@ -890,11 +890,7 @@ public function redirect($url, $statusCode = 302, $checkAjax = true)
                     // Ajax 302 redirect in IE does not work. Change status code to 200. See https://github.com/yiisoft/yii2/issues/9670
                     $statusCode = 200;
                 }
-                if (Yii::$app->getRequest()->getIsPjax()) {
-                    $this->setHeader('X-Pjax-Url', $url);
-                } else {
-                    $this->setHeader('X-Redirect', $url);
-                }
+                $this->setHeader('X-Redirect', $url);
```

Comment:
I think this one was for PJAX as well.

Reply:
Why?
The original condition was:

```
if (Yii::$app->getRequest()->getIsPjax()) {
    $this->setHeader('X-Pjax-Url', $url);
} else {
    $this->setHeader('X-Redirect', $url);
}
```

check for AJAX is located above that.

Reply:
Ah, OK.

Reply:
http://www.yiiframework.com/doc-2.0/yii-web-response.html#redirect()-detail